app/react/Metadata/helpers/formater.js

```diff
@@ -45,9 +45,22 @@ const addCreationDate = (result, doc) =>
     sortedBy: true,
   });
 
+const addModificationDate = (result, doc) =>
+  result.push({
+    value: moment.utc(doc.creationDate).format('ll'),
+    label: 'Date modified',
+    name: 'editDate',
+    translateContext: 'System',
+    sortedBy: true,
+  });
+
```

Comment:
@grafitto this is not tested, actually the value being formated is wrong, i think "creationDate" should be "editDate" ? it is really usefull to test first to make you think about the implementation and avoid this kind of errors.

Reply:
Added tests and made the update, nice catch.